app/models/accession_service.rb

```diff
@@ -19,6 +19,14 @@ class AccessionedFile < File
     attr_accessor :original_filename
   end
 
+  class_attribute :priority, :instance_writer => false
+  class_attribute :no_study_accession_needed, :instance_writer => false
+  class_attribute :operational, :instance_writer => false
+
```

Comment:
Maybe we can add a comment for priority and operational to remember what they are used for. Priority is related with the priorities of other accession services and operational means that they are currently active, right?